doc/release_notes.md

```diff
@@ -0,0 +1,21 @@
+<!--
+# SPDX-FileCopyrightText:  Open Energy Transition gGmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+# Features and Contributions
+
+Please list contributions, add reference to PRs if present.
+
+* Analyzed statewise **passengers and fuel consumption data** for fuel demand disaggregation for airports: [PR #9](https://github.com/open-energy-transition/efuels-supply-potentials/pull/9) 
+
+* Facilitated **PyPSA-Earth sector run**: PyPSA-Earth [PR #1134](https://github.com/pypsa-meets-earth/pypsa-earth/pull/1134), [PR #1143](https://github.com/pypsa-meets-earth/pypsa-earth/pull/1143), [PR #1145](https://github.com/pypsa-meets-earth/pypsa-earth/pull/1145), [PR #1165](https://github.com/pypsa-meets-earth/pypsa-earth/pull/1145), [PR #1166](https://github.com/pypsa-meets-earth/pypsa-earth/pull/1166)
```

Comment:
Hi @yerbol-akhmetov, there is a duplicate entry here for PR 1145.

Reply:
> Hi @yerbol-akhmetov, there is a duplicate entry here for PR 1145.

Hi, Emmanuel . Thanks. I think there is a typo. The second one should be 1165 as in its url. I will change it and let you know.